src/pt/spectralscan/src/eu/kanade/tachiyomi/extension/pt/spectralscan/NexusScan.kt

```diff
@@ -0,0 +1,311 @@
+                    label.contains("artista") || label.contains("artist") -> artist = value
+                    label.contains("status") -> {
+                        status = when (value?.lowercase()) {
+                            "em andamento", "ongoing", "ativo", "lançando" -> SManga.ONGOING
+                            "completo", "completed", "finalizado" -> SManga.COMPLETED
+                            "pausado", "hiato", "on hiatus" -> SManga.ON_HIATUS
+                            "cancelado", "cancelled", "dropped" -> SManga.CANCELLED
+                            else -> SManga.UNKNOWN
+                        }
+                    }
+                }
+            }
+        }
+    }
+
+    // ==================== Chapter =======================
+
+    private fun getMangaSlug(url: String) = url.substringAfter("/manga/").trimEnd('/')
+
+    override fun fetchChapterList(manga: SManga): Observable<List<SChapter>> {
+        return client.newCall(chapterListRequest(manga)).asObservableSuccess()
+            .map { it.parseAs<ChapterListResponse>() }
+            .flatMap { firstPage ->
+                val chapters = parseChaptersFromHtml(firstPage.chapters_html)
+
+                if (!firstPage.has_next) {
+                    Observable.just(chapters)
+                } else {
+                    fetchRemainingChapters(getMangaSlug(manga.url), chapters, 2)
+                }
+            }
+    }
+
+    private fun fetchRemainingChapters(
+        slug: String,
+        accumulatedChapters: List<SChapter>,
+        currentPage: Int,
+    ): Observable<List<SChapter>> {
+        val url = "$baseUrl/ajax/load-chapters/".toHttpUrl().newBuilder()
+            .addQueryParameter("item_slug", slug)
+            .addQueryParameter("page", currentPage.toString())
+            .addQueryParameter("sort", "desc")
+            .addQueryParameter("q", "")
+            .build()
+
+        return client.newCall(GET(url, headers)).asObservableSuccess()
+            .map { it.parseAs<ChapterListResponse>() }
+            .flatMap { page ->
+                val allChapters = accumulatedChapters + parseChaptersFromHtml(page.chapters_html)
+
+                if (page.has_next) {
+                    fetchRemainingChapters(slug, allChapters, currentPage + 1)
+                } else {
+                    Observable.just(allChapters)
+                }
+            }
+    }
+
+    private fun parseChaptersFromHtml(html: String): List<SChapter> {
+        val document = Jsoup.parse(html, baseUrl)
+        return document.select("a.chapter-item[href]").map { element ->
+            SChapter.create().apply {
+                name = element.selectFirst(".chapter-number")?.text()?.trim()
+                    ?: element.attr("data-chapter-number")?.let { "Capítulo $it" }
+                    ?: "Capítulo"
+                setUrlWithoutDomain(element.absUrl("href").ifEmpty { element.attr("href") })
```

Comment:
You shouldn't need `.ifEmpty` if `element` knows what page its on and knows how to resolve relative paths.
```suggestion
                setUrlWithoutDomain(element.absUrl("href"))
```